pkg/datasource/executor.go

```diff
@@ -0,0 +1,61 @@
+package datasource
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/grafana/influxdb-flux-datasource/pkg/models"
+	influxdb2 "github.com/influxdata/influxdb-client-go"
+)
+
+// QueryResultToDataFrames creates a DataFrame from query results
+func GetRawFluxQuery(query *models.QueryModel) (string, error) {
+	// TODO apply macros!!!
+
+	return query.RawQuery, nil
+}
+
+func ExecuteQuery(ctx context.Context, query *models.QueryModel, runner queryRunner) (dr *backend.DataResponse) {
```

Comment:
`dr *backend.DataResponse` to `dr backend.DataResponse`. Then in `QueryData` `res.Responses[q.RefID] = *d` becomes `res.Responses[q.RefID] = d`